engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeExecutionEnvironmentTest.scala

```diff
@@ -129,8 +129,15 @@ class RuntimeExecutionEnvironmentTest
   }
 
   override protected def afterEach(): Unit = {
-    context.executionContext.context.close()
-    context.runtimeServerEmulator.terminate()
+    if (context != null) {
+      context.reset()
```

Comment:
This sequence seems to appear at many places. Shouldn't it be shared at a single place?